Api Web/Service/UserRepo.cs

```diff
@@ -66,6 +67,7 @@ public async Task<User> GetUser(int id)
         public async Task<User> updateUser(User userUpdate)
         {
             _dataContext.Update(userUpdate);
+
             await _dataContext.SaveChangesAsync();
```

Comment:
Same here, validate if the item updated was updated successfully, you may want to add try-catch bloc scope or use a condition to make sure the changes were saved correctly.